### ISSUE TITLE
[Feat]: add support for --disable-extensions

### ISSUE THREAD

**Issue by niole** [coder/code-server#6722]
## What is your suggestion?
- add support for --disable-extensions to `code-server`

## Why do you want this feature?
I work at a company that bootstraps development environments with vscode. Our customers are security conscious and have found that certain extensions contain security vulnerabilities. We want to permanently disable these extensions.

## Are there any workarounds to get this functionality today?
Not that I know of

## Are you interested in submitting a PR for this?
Yes

**Comment by matuskosut** [comment]
I agree, very important for those that don't allow end users to install extensions themselves.

This would be very nice feature especially for adding extensions that complain about requiring initial configuration (e.g. path to executable) when Code server is opened. These extensions would typically be needed only for small number of users so it would be great to disable them during docker build, and let users enable when they want.

**Comment by gogo25171** [comment]
Hey, VS Code has a native feature for managing extensions from the command line:
https://code.visualstudio.com/docs/configure/command-line#_working-with-extensions

If you add support for this feature as launch arguments in code-server, could you also add an argument like:

"--disable-extensions-except ID1 ID2 ID3"

This would allow users to disable all extensions except a specific list of extension IDs.
